### DISCUSSION TITLE
Beware of Wolves in Sheep's Clothing

### DISCUSSION THREAD

**Discussion by tteck** [community-scripts/ProxmoxVE#18] [Announcements]
It only takes one person to disrupt trust and create chaos. Being discerning about who we allow into our circle is crucial.

**Comment by kissdisap** [comment]
Thank you for all you've done for the community tteck. My Proxmox installation wouldn't be where it is without your contributions! Wishing you all the best

**Comment by Dvalin21** [comment]
Thanks for the wisdom

> It only takes one person to disrupt trust and create chaos. Being discerning about who we allow into our circle is crucial.

Thank you so much for your wisdom and knowledge!!

**Comment by newzealandpaul** [comment]
I hope the community heeds @tteck's warning.

@tteck I don't know where in the world you are but I wish we had the opportunity to meet in real life.

**Comment by MaskUp** [comment]
Thank you tteck, I've used these scripts more times then I can count. It's the first thing I google when I hear about another self hosted app that sounds amazing, only to find a one line proxmox script that builds it for me.

May you be at peace, your imprint will continue

**Comment by txelu1997** [comment]
Thank you very much for your excellent work on this planet!
I am already looking forward to seeing your next works on other planets in the ages to come.

"47:3.1 (532.7)On the mansion worlds the resurrected mortal survivors resume their lives just where they left off when overtaken by death. When you go from Urantia to the first mansion world, you will notice considerable change, but if you had come from a more normal and progressive sphere of time, you would hardly notice the difference except for the fact that you were in possession of a different body; the tabernacle of flesh and blood has been left behind on the world of nativity.

47:3.2 (532.8)The very center of all activities on the first mansion world is the resurrection hall, the enormous temple of personality assembly. This gigantic structure consists of the central rendezvous of the seraphic destiny guardians, the Thought Adjusters, and the archangels of the resurrection. The Life Carriers also function with these celestial beings in the resurrection of the dead.

47:3.3 (533.1)The mortal-mind transcripts and the active creature-memory patterns as transformed from the material levels to the spiritual are the individual possession of the detached Thought Adjusters; these spiritized factors of mind, memory, and creature personality are forever a part of such Adjusters. The creature mind-matrix and the passive potentials of identity are present in the morontia soul intrusted to the keeping of the seraphic destiny guardians. And it is the reuniting of the morontia-soul trust of the seraphim and the spirit-mind trust of the Adjuster that reassembles creature personality and constitutes resurrection of a sleeping survivor."
